integration/client/restart_monitor_linux_test.go

```diff
@@ -102,8 +107,12 @@ version = 2
 		}
 		time.Sleep(epsilon)
 	}
-	t.Fatalf("%v: the task was not restarted in %s + %s",
-		time.Now(), interval.String(), epsilon.String())
+
+	now := time.Now()
+	if now.After(begin.Add(interval).Add(epsilon)) {
```

Comment:
What is the reason for checking this here? The only success case will return in the loop

Reply:
I've changed the deadline from `interval + epsilon` to `interval * 2`. 

If the restart is a few seconds off, the test can catch that in this if.